app/main/helpers/frameworks.py

```diff
@@ -252,4 +252,4 @@ def get_status_for_multi_service_lot_and_service_type(
 def has_one_service_limit(lot_slug, framework_lots):
     for lot in framework_lots:
         if lot['slug'] == lot_slug:
-            return lot['one_service_limit']
+            return lot['oneServiceLimit']
```

Comment:
Is this changing in a breaking way? I’ve been using the :snake: :briefcase: one in a few places :disappointed:.